tests/python/relax/test_transform_rewrite_cuda_graph.py

```diff
@@ -0,0 +1,87 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import tvm
+from tvm import relax
+from tvm.script import tir as T, relax as R
+from tvm import topi
+import numpy as np
+from tvm.ir import IRModule
+import tvm.testing
+import tempfile
+import tvm.relax.transform.tuning_api.default_functions
+from tvm.relax.transform.tuning_api import Trace
+
+def apply_tuning_passes(mod, target):
+    with tempfile.TemporaryDirectory() as work_dir:
+        with target, tvm.transform.PassContext(trace=Trace(mod)):
+            seq = tvm.transform.Sequential(
+                [
+                    relax.transform.MetaScheduleTuneIRMod(
+                        params={}, work_dir=work_dir, max_trials_global=8
+                    ),
+                    relax.transform.MetaScheduleApplyDatabase(work_dir),
+                ]
+            )
+            mod = seq(mod)
+    return mod
+
+
+@tvm.testing.requires_cuda
+def test_minimum_example():
```

Comment:
would be nice to have UT here that compares before/after like other transform tests.

Then we can have test_runtime_cuda_graph.py that tests the runtime capabilities along with compilation